src/QCToolsBundle/Resources/views/Download/macTable.html.twig

```diff
@@ -10,9 +10,14 @@
     <td colspan="5">&nbsp;<br>&nbsp;</td>
 </tr>
 <tr>
-    <th rowspan="2"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
-    <th rowspan="2">x86_64</th>
-    <td><abbr title="Graphical User Interface">GUI</abbr></td>
+    <th rowspan="3"><br>10.10&nbsp;(Yosemite),<br>10.11&nbsp;(El&nbsp;Captain),<br>10.12&nbsp;(Sierra),<br>10.13&nbsp;(High&nbsp;Sierra),<br>10.14&nbsp;(Mojave)<br>10.15&nbsp;(Catalina)</th>
```

Comment:
10.10 and 10.11 not compatible.
Also in the remarks part (right column), something like "Include support of dark theme on macOS 10.14 (Mojave) onward".